src/property-filter/__tests__/property-filter.test.tsx

```diff
@@ -303,12 +308,54 @@ describe('property filter parts', () => {
         expect.objectContaining({ filteringText: '123', filteringProperty: stateProperty, filteringOperator: '=' }),
       ]);
 
+      createWrapper(container).find('#focus-target')!.focus();
+
       wrapper.findClearButton()!.click();
       jest.advanceTimersByTime(1000);
+
       expect(loadItemCalls).toEqual([
         expect.objectContaining({ filteringText: '' }),
         expect.objectContaining({ filteringText: '' }),
         expect.objectContaining({ filteringText: '123', filteringProperty: stateProperty, filteringOperator: '=' }),
+        expect.objectContaining({ filteringText: '123', filteringProperty: stateProperty, filteringOperator: '=' }),
+        expect.objectContaining({ filteringText: '' }),
+        expect.objectContaining({ filteringText: '' }),
+      ]);
+    });
+
+    test('calls onLoadItems on clear when filter by text and asyncProperties=true', () => {
+      const loadItemCalls: PropertyFilterProps.LoadItemsDetail[] = [];
+      const { propertyFilterWrapper: wrapper, container } = renderComponent({
+        onLoadItems: ({ detail }) => loadItemCalls.push(detail),
+        filteringStatusType: 'pending',
+        asyncProperties: true,
+      });
+
+      wrapper.focus();
+      expect(loadItemCalls).toEqual([
+        expect.objectContaining({ filteringText: '' }),
+        expect.objectContaining({ filteringText: '' }),
+      ]);
+
+      wrapper.setInputValue('stat');
+      expect(loadItemCalls).toHaveLength(2);
+
+      jest.advanceTimersByTime(1000);
+      expect(loadItemCalls).toEqual([
+        expect.objectContaining({ filteringText: '' }),
+        expect.objectContaining({ filteringText: '' }),
+        expect.objectContaining({ filteringText: 'stat' }),
+      ]);
+
+      createWrapper(container).find('#focus-target')!.focus();
+
+      wrapper.findClearButton()!.click();
+
+      expect(loadItemCalls).toEqual([
+        expect.objectContaining({ filteringText: '' }),
+        expect.objectContaining({ filteringText: '' }),
+        expect.objectContaining({ filteringText: 'stat' }),
+        expect.objectContaining({ filteringText: 'stat' }),
         expect.objectContaining({ filteringText: '' }),
```

Comment:
What is different between free-text and parsed option? Since we receive the expected load-more call for the latter, that should be possible for the free-text, too, but w/o causing the second call of it in the other test.

Reply:
this is actually a matter of handling `filteringStatusType`. If we update the status to `finished` in the example with parsed option, the desired event won't be called either